worker/src/lib.rs

```diff
@@ -162,6 +162,7 @@ mod cors;
 pub mod d1;
 mod date;
 mod delay;
+pub mod digest_stream;
```

Comment:
Typically, we would re-export this. I'm not sure that I like this pattern, so we can keep it like this, but I think the module name should be `crypto`.